examples/CMakeLists.txt

```diff
@@ -1,4 +1,38 @@
-option(PENGUINV_BUILD_QT_EXAMPLE "Build Qt examples of penguinV" OFF)
-if(${PENGUINV_BUILD_QT_EXAMPLE})
-    add_subdirectory(qt_framework)
+# Link this 'library' to set the c++ standard / compile-time options requested
+add_library(example_features_options INTERFACE)
+target_compile_features(example_features_options INTERFACE cxx_std_11)
+
+# Link this 'library' to use the standard warnings
+add_library(example_features_warnings INTERFACE)
+if(MSVC)
+    target_compile_options(example_features_warnings
+                           INTERFACE /W4 /WX)
+else()
+    target_compile_options(example_features_warnings
+                           INTERFACE -Wall
+                                     -Wextra           # reasonable and standard
+                                     -Wstrict-aliasing # warn the user if a variable declaration shadows one
+                                     -Wpedantic        # warn if non-standard C++ is used
+                                     -Wconversion
+                                     -Wsign-conversion
+                                     -O2
+                                     -march=native)
 endif()
+
+set(LIB_DIR ${CMAKE_SOURCE_DIR}/src)
+add_subdirectory(bitmap_operation)
+add_subdirectory(blob_detection)
+if(CUDA_FOUND)
```

Comment:
Let's remove this if else with cuda examples as of now or just comment these files

Reply:
OK